portal/server/routes/login/index.js

```diff
@@ -88,7 +67,7 @@ router.post('/login', async (req, res) => {
     console.warn('Failed to login: %O', loginError);
 
     if (loginError.response?.status === 403) {
-      return res.render('login/temporarily-suspended.njk');
+      return res.status(403).render('login/temporarily-suspended.njk');
```

Comment:
I've added this to return a 403 as there's existing precident for returning correct statuses on failed requests (ie delete account)